syncano_cli/parse_to_syncano/migrations/transfer.py

```diff
@@ -97,6 +98,8 @@ def transfer_objects(self, instance):
                     objects_to_add.append(batched_syncano_object)
                     parse_ids.append(data_object['objectId'])
 
+                    time.sleep(6)  # avoid throttling;
```

Comment:
Hmm, because the batch make 10 creates/updates 60/10 = 6 - or am I bad at math? :)